lib/components/Information/Avatar/index.tsx

```diff
@@ -1 +1,23 @@
-export * from "@/ui/avatar"
+import {
+  Avatar as AvatarComponent,
+  AvatarFallback,
+  AvatarImage,
+} from "@/ui/avatar"
+import { forwardRef } from "react"
+
+interface AvatarType {
+  alt: string
+  src?: string
+  size?: "medium" | "small" | "xsmall" | "large" | "xlarge" | "xxlarge"
```

Comment:
Should we follow tailwind's conventions? They usually use `md`, `sm`, `xs`, `lg`, `xl` and `2xl`. Not that important, I guess.